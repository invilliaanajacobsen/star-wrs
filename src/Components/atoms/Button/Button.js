import React from 'react';
import { ButtonCustom } from './style.js';


const Button = ({ text }) => (
	<ButtonCustom>{text}</ButtonCustom>
);

export default Button;