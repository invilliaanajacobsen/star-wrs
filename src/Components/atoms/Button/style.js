import styled from 'styled-components';
import Button from 'react-bootstrap/Button'


export const ButtonCustom = styled(Button)`
	text-transform: uppercase;
	line-height: 1.3rem;
	border: none;
	border-radius: 6px;
	background-color: #737373;

	:hover {
		background-color: #fff;
		color: #565656;
	  }
`;

