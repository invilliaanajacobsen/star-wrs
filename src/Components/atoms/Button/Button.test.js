import React from "react";
import { shallow, mount } from "enzyme";

import Button from "./Button";

describe("Button component", () => {
  it("should render Button match snapshot", () => {
    expect(shallow(<Button />)).toMatchSnapshot();
  });

  it('should render Button with proprety text', () => {
    expect(shallow(<Button text="botao" />)).toMatchSnapshot();
    expect(mount(<Button text="botao" />).find("button").length).toBe(1);
  });
});
