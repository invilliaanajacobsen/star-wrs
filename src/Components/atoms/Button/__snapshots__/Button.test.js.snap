// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render Button match snapshot 1`] = `<Styled(Button) />`;

exports[`Button component should render Button with proprety text 1`] = `
<Styled(Button)>
  botao
</Styled(Button)>
`;
